.github/workflows/lint.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
   flake8_py3:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@main
```

Comment:
I think this was actually referring to the branch name of https://github.com/actions/checkout? Anyhow doesn't seem to have broken anything

Reply:
Yes, that was intended actually, mentioned in the original PR that got automatically closed with the rename :/

(#157)

Thanks for pointing it out though :+1: